app/assets/javascripts/router.js

```diff
@@ -7,7 +7,8 @@
   root.App.Router = Backbone.Router.extend({
 
     routes: {
-      'case-studies/:id?page=:page': 'caseStudies'
+      'case-studies/:id?page=:page': 'caseStudies',
+      'case-studies/:id': 'caseStudies'
     },
```

Comment:
You can add (?page=:page) so its an optional parameter, that way you won't have to repeat the same line =)